ABB.SrcML/LibSrcMLRunner.cs

```diff
@@ -1,10 +1,12 @@
-﻿using System;
+﻿/* Setup options for srcml unit */
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Runtime.InteropServices;
 namespace ABB.SrcML {
-    public class SrcMLCppAPI {
+    public class LibSrcMLRunner {
```

Comment:
Consider applying the [`CLSCompliant(false)`](https://msdn.microsoft.com/en-us/library/12a7a7h3%28v=vs.120%29.aspx#CLSAttribute) attribute to `LibSrcMLRunner` -- this will clear up a bunch of the